skills/cmux-debug-windows/SKILL.md

```
@@ -23,9 +23,8 @@ Keep this workflow focused on existing debug windows and menu entries. Do not ad
 ```bash
 skills/cmux-debug-windows/scripts/debug_windows_snapshot.sh --copy
 ```
-5. After code edits, run build + tagged reload:
+5. After code edits, run the tagged reload:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)